charts/cass-operator-chart/templates/customresourcedefinition.yaml

```diff
@@ -1316,7 +1316,9 @@ spec:
                             {{- if semverCompare ">= 1.16-0" .Capabilities.KubeVersion.GitVersion }}
                             x-kubernetes-list-map-keys:
                             - containerPort
+                            {{- if semverCompare "< 1.18-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
this is the "fix" - k8s 1.18 validation asserts that keys in this list are required or have a default value, but the full wiring to those defaults is broken for core types right now - see https://github.com/kubernetes-sigs/controller-tools/issues/444